From 434727141736288f3eb3058897817e34f9c2c74a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:00 -0400
Subject: [redhat] kabi: Adding symbol scsi_host_lookup (drivers/scsi/hosts.c)

Message-id: <3d3c1650931e767f730ab41b67c060908a7212a2.1410510773.git.poros@redhat.com>
Patchwork-id: 96074
O-Subject: [PATCH RHEL7.1 180/233] kabi: Adding symbol scsi_host_lookup (drivers/scsi/hosts.c)
Bugzilla: 1071099
RH-Acked-by: Rob Evers <revers@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Get a reference to a Scsi_Host by host no.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/scsi_host_lookup
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/scsi_host_lookup
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/scsi_host_lookup

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/scsi_host_lookup b/redhat/kabi/kabi-module/kabi_ppc64/scsi_host_lookup
new file mode 100644
index 0000000..f624897
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/scsi_host_lookup
@@ -0,0 +1,2 @@
+#1-
+0xb4a97c80 scsi_host_lookup vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/scsi_host_lookup b/redhat/kabi/kabi-module/kabi_s390x/scsi_host_lookup
new file mode 100644
index 0000000..eada12b
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/scsi_host_lookup
@@ -0,0 +1,2 @@
+#1-
+0xb8d661af scsi_host_lookup vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/scsi_host_lookup b/redhat/kabi/kabi-module/kabi_x86_64/scsi_host_lookup
new file mode 100644
index 0000000..66b2255
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/scsi_host_lookup
@@ -0,0 +1,2 @@
+#1-
+0xb3839e3b scsi_host_lookup vmlinux EXPORT_SYMBOL
-- 
1.7.1